.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -476,6 +476,13 @@ const getPipeline = (filename: string, removeSteps = true) => {
       );
     }
 
+    if (GITHUB_PR_LABELS.includes('ci:security-genai-prompts-run-evals')) {
+      process.env.IS_PROMPT = 'true';
```

Comment:
@spong can I set an env variable right here? Would be handy!

Reply:
I would recommend something like https://github.com/elastic/kibana/blob/9566f1a7b3aa9d17200a0d1a57eb20f096497e79/.buildkite/scripts/common/env.sh#L77-L81

This could cause CI to approve pull requests that will fail in on-merge.  We're doing this in a few other areas and I'm not overly concerned, but something to keep in mind.